tests/pyqtslot.py

```diff
@@ -8,9 +10,84 @@ def func_none(s: str) -> None:
 def func_int(s: str) -> int:
     return 42
 
-@pyqtSlot(str, result='int')
+@pyqtSlot(str, result='int')    # this means int here
 def func_str(s: str) -> str:
```

Comment:
Is this testing incorrect usage?  I would expect that `result='int'` would correspond with `-> int:`.  Though I've never really dealt much with slot return values.

Reply:
This was introduced long before I joined the project. I agree it's hard to interpret correctly, that's certainly what the initial author wanted to test.